skglm/datafits/group.py

```diff
@@ -50,6 +51,20 @@ def get_lipschitz(self, X, y):
 
         return lipschitz
 
+    def get_lipschitz_sparse(self, X_data, X_indptr, X_indices, y):
+        grp_ptr, grp_indices = self.grp_ptr, self.grp_indices
+        n_groups = len(grp_ptr) - 1
+        lipschitz = np.zeros(n_groups, dtype=X_data.dtype)
+
+        X = sparse.csc_matrix((X_data, X_indices, X_indptr))
+        for g in range(n_groups):
+            grp_g_indices = grp_indices[grp_ptr[g]: grp_ptr[g+1]]
+            X_g = X[:, grp_g_indices]
+            lipschitz[g] = sparse.linalg.norm(X_g, ord=2) ** 2 / len(y)
```

Comment:
computation on matrix 2-norm on csc is not supported by numba, any ideas how to address this?

Reply:
we have a dedicated function: https://github.com/scikit-learn-contrib/skglm/blob/main/skglm/utils/sparse_ops.py#L7